cli/tests/unit/mkdir_test.ts

```diff
@@ -197,3 +197,31 @@ unitTest(
     }
   },
 );
+
+unitTest(
+  { perms: { read: true, write: true } },
+  function mkdirSyncRelativeUrlPath(): void {
+    const testDir = Deno.makeTempDirSync();
+    const nestedDir = testDir + "/nested";
+    const path = new URL(nestedDir + "../dir");
```

Comment:
Should try `const path = new URL("../dir", nestedDir);`

Reply:
Thanks!